packages/client-search/src/types/ApiKeyType.ts

```diff
@@ -7,6 +7,7 @@ export const ApiKeyACLEnum: Readonly<Record<string, ApiKeyACLType>> = {
   EditSettings: 'editSettings',
   ListIndexes: 'listIndexes',
   Logs: 'logs',
+  Personalization: 'personalization',
   Recommendation: 'recommendation',
```

Comment:
is one of these two deprecated?

Reply:
I guess the two will work for some time. Wdym?

Reply:
good enough for now!